src/jquery.countdown360.js

```diff
@@ -27,7 +27,16 @@
   }
 
   Plugin.prototype = {
-
+    getTimeRemaining: function()
+    {
+      var elapsed = this.getElapsedTime();
+      var timeRemaining = this._secondsLeft(elapsed);
```

Comment:
Can you combine these two var statements? I have fairly strict build checking and I don't want to change it. See lines 41-42 for an example of what I mean. Thanks :smile_cat: